tests/nested/core/save-data/task.yaml

```diff
@@ -0,0 +1,44 @@
+summary: Run a smoke test on UC with encryption enabled
+
+details: |
+    This test checks basic snapd commands on UC with secure boot and encryption enabled
```

Comment:
the description here seems to need updating/clarifying

Reply:
Fixed!